CONTRIBUTING.md

```diff
@@ -1,11 +1,86 @@
 
-All contributors sending Pull Requests (PRs) must have a Contributor License Agreement 
-on file as either an [individual](https://www.cloudfoundry.org/pdfs/CFF_Individual_CLA.pdf) 
-or via their [employer](https://www.cloudfoundry.org/pdfs/CFF_Corporate_CLA.pdf).
+If there is a need for some discussion around how best to
+address the concern then opening an Issue, prior to doing the work to develop
+a PR, would be best.  These minor issues do not need a formal review, per the
+[process](#prissue-review-process) described below, rather the issue
+should be used to come to a conscensus around a PR that will eventually be
+submitted.
+
+If the proposed change is non-controversial (e.g. a typo) then a PR can be
+submitted directly without opening an issue.
+
+Either way, once a PR is submitted it will be reviewed per the
+[process](#prissue-review-process) described below.
+
+## Design Change Proposals
+
+New design proposals to the API spec should be submitted by opening a
+Github issue with a link to a Google Doc containing the proposal. Proposals
+should contain motivation, use cases, and detailed example of proposed
+changes to API interactions. Collaboration on the design, fleshing out of
+use cases, etc can occur as comment discussions in the Google Doc, as well
+as on our weekly calls.
+
+For changes largely impacting the SB API actors (platforms, service authors,
+etc...), it is recommended to solicit feedback from these actors and leave
+enough time (say 2 weeks) for feedback to be provided, and for the
+potentially received objections/suggestions to be handled.
+
+Once the design has been finalized in the Google Doc, the proposed set of
+changes to the specification should be made available for review. This could
+be done by pointing to a branch in a Github repo with the proposed edits.
+Reviewers, or implementers of the feature, can then easily see the exact
+changes being proposed and comment on them within that branch.
+
+Before a formal PR for the changes is created, the feature needs to be
+validated through a full implementation by at least one platform that is
+easily accessible to reviewers. Note that in many cases this would include
+support for the proposed changes by one or more service brokers
+(potentially samples). How each platform makes this new feature availble
+will vary between platforms. It is strongly recommended that as part of the
+design review process platforms that plan on implementing the feature
+get confirmation from the reviewers that their mechanims for providing
+access to the implementation of the feature is acceptable.
+
+Once support for the change has been implemented, a PR should be sent to
+this repo for the change to the broker API specification. By this point,
+the API interactions should be well understood and there should be no
+technical surprises; we expect the only discussion necessary on PRs to be
+for wordsmithing and formatting. It is expected that during this implementation
```

Comment:
> It is expected that during this implementation
> phase there will be edits made to the design and proposed specification
> to accurately represent the current status of the proposal.

I'd suggest this setence should be moved up a bit prior to "Once support for the change has been implemented, a PR should be sent to ...", as to clarify edits to design and specs are done through the "validation" phase